components/sidebar/sidebar_channel/sidebar_channel_link/sidebar_channel_link.tsx

```diff
@@ -132,18 +137,46 @@ export default class SidebarChannelLink extends React.PureComponent<Props, State
 
     render() {
         const {link, label, channel, unreadMentions, icon, isMuted} = this.props;
+
+        let hoverLabel: JSX.Element = (
```

Comment:
It's necessary to allow the tooltip to display correctly.
The `div` is there because the tooltip wouldn't display correctly on the desktop without it.
The `span` is there to wrap the message so that it can be cutoff correct and measured to enable the tooltip.